src/routers/login.routes.ts

```diff
@@ -0,0 +1,26 @@
+import {NextFunction, Request, Response} from 'express';
+import * as express from "express";
+import {loginValidator} from "../validators/validator";
+import {LoginService} from '../service/login.service';
+import jwt from "jsonwebtoken";
+import {JWT_SECRET_KEY} from "../constants/constants";
+
+const validator = require('express-joi-validation').createValidator({});
+const service = new LoginService();
+export const loginRouter = express.Router();
+
+loginRouter.get('/', validator.body(loginValidator), (req: Request, res: Response, next: NextFunction) => {
+  const { login, password } = req.body;
```

Comment:
No business logic in controller, please move everything to servies.

Reply:
done